### ISSUE TITLE
Unable to get time with NVDA+F12

### ISSUE THREAD

**Issue by LeonarddeR** [nvaccess/nvda#18959]
### Brief summary

It is not possible to get the time with NVDA's get current time/date script (NVDA+F12)

### Steps to reproduce

Press NVDA+F12

### Actual behavior

An error sound is heard and the time is not spoken. Note that a second press is able to report the date.

### Expected behavior

NO error, time is spoken

### NVDA logs, crash dumps and other attachments

```log
ERROR - scriptHandler.executeScript (07:40:22.102) - MainThread (2816):
error executing script: <bound method GlobalCommands.script_dateTime of <globalCommands.GlobalCommands object at 0x06B81538>> with gesture 'NVDA+f12'
Traceback (most recent call last):
  File "scriptHandler.pyc", line 300, in executeScript
  File "globalCommands.pyc", line 442, in script_dateTime
  File "winKernel.pyc", line 331, in GetTimeFormatEx
ctypes.ArgumentError: argument 2: TypeError: 'NoneType' object cannot be interpreted as an integer
```

### NVDA type

installed copy

### NVDA version

alpha-52705,dc226976 (2026.1.0.52705)

### Have you tried any other versions of NVDA? If so, please report their behaviors.

Yes, beta is not affected

### Windows version

Windows 11 25H2

### Does the issue still occur after restarting your computer?

I have not restarted my computer

### If NVDA add-ons are disabled, is your problem still occurring?

I have not restarted NVDA with add-ons disabled

### Does the issue still occur after you run the COM Registration Fixing Tool in NVDA's tools menu?

I have not run the COM Registration Fixing Tool

**Comment by LeonarddeR** [comment]
Here is a more detailed traceback when running from source:
```log
error executing script: <bound method GlobalCommands.script_dateTime of <globalCommands.GlobalCommands object at 0x08343E90>> with gesture 'NVDA+f12'
Traceback (most recent call last):
  File "scriptHandler.py", line 300, in executeScript
    script(gesture)
    ~~~~~~^^^^^^^^^
  File "globalCommands.py", line 442, in script_dateTime
    text = winKernel.GetTimeFormatEx(winKernel.LOCALE_NAME_USER_DEFAULT, None, None, None)
  File "winKernel.py", line 331, in GetTimeFormatEx
    bufferLength = winBindings.kernel32.GetTimeFormatEx(Locale, dwFlags, lpTime, lpFormat, None, 0)
ctypes.ArgumentError: argument 2: TypeError: 'NoneType' object cannot be interpreted as an integer
```

**Comment by SaschaCowley** [comment]
Hi @LeonarddeR, I can't repro this on my system, presumably because I have my system clock set to hide seconds. Could you see if #18960 fixes your issue?

**Comment by LeonarddeR** [comment]
I indeed have second reporting enabled.
https://github.com/nvaccess/nvda/pull/18960 fixes it for me, thanks!
